py-polars/polars/dataframe/frame.py

```diff
@@ -4122,6 +4122,120 @@ def sort(
             .collect(_eager=True)
         )
 
+    def sql(self, query: str, table_name: str | None = None) -> Self:
```

Comment:
Maybe `table_name` should be a keyword-only argument?

Reply:
Sounds reasonable; can update later tonight 👌